server/system/rest.yaml

```diff
@@ -2256,3 +2256,34 @@ endpoints:
             required: false
             title: Exclude (0, default), include (1) or return only (2) deleted connections
             type: filter.State
+
+- title: SMTP Connection Checker
+  entrypoint: smtpConnection
+  path: "/smtp/connection"
+  apis:
+    - name: check
+      method: POST
+      title: Check SMTP server settings
+      path: "/checker/"
+      parameters:
```

Comment:
did you skip other connection options (tls insecure, tls server name) intentionally?